tools/depends/target/python3/01-win-cmake.patch

```diff
@@ -0,0 +1,1140 @@
+--- /dev/null
++++ b/cmake/python-config.cmake
+@@ -0,0 +2 @@
++include(${CMAKE_CURRENT_LIST_DIR}/python.cmake)
++
+--- /dev/null
++++ b/CMakeLists.txt
+@@ -0,0 +1,1132 @@
++cmake_minimum_required(VERSION 3.15)
++
++project(python VERSION 3.11.7 LANGUAGES C CXX)
++
++find_package(zlib NO_MODULE REQUIRED HINTS ${DEPENDS_PATH}/lib)
++find_package(expat REQUIRED HINTS ${DEPENDS_PATH}/lib)
++find_package(openssl 1.1.1 NO_MODULE REQUIRED HINTS ${DEPENDS_PATH}/lib)
++find_package(bzip2 NO_MODULE REQUIRED HINTS ${DEPENDS_PATH}/lib)
++find_package(libffi NO_MODULE REQUIRED HINTS ${DEPENDS_PATH}/lib)
++find_package(xz NO_MODULE REQUIRED HINTS ${DEPENDS_PATH}/lib)
++find_package(sqlite3 NO_MODULE REQUIRED HINTS ${DEPENDS_PATH}/lib)
++
++include(findpackagehandlestandardargs)
++# pythoncore/python3dll
++find_package_handle_standard_args(zlib CONFIG_MODE)
++# modules
++find_package_handle_standard_args(expat CONFIG_MODE)
++find_package_handle_standard_args(openssl CONFIG_MODE)
++find_package_handle_standard_args(bzip2 CONFIG_MODE)
++find_package_handle_standard_args(libffi CONFIG_MODE)
++find_package_handle_standard_args(xz CONFIG_MODE)
++find_package_handle_standard_args(sqlite3 CONFIG_MODE)
++
++# Based on PCbuild\find_python.bat that allows us to cache to NATIVEPREFIX
++add_custom_target(script.find_python
++    COMMAND powershell -Command Invoke-WebRequest https://aka.ms/nugetclidl -OutFile '${NATIVEPREFIX}/bin/nuget.exe'
```

Comment:
```suggestion
+    COMMAND @PowerShell "Invoke-WebRequest https://aka.ms/nugetclidl -OutFile '${NATIVEPREFIX}/bin/nuget.exe'"
```

Reply:
This fix the first issue, the rest I have no idea for now.

Reply:
Greatly appreciated. I'll look into it.

What VS are you using, and is it a specific SDK to install over the "default" for that VS version?
Also, was this for UWP or the Desktop variant (or both)?

Reply:
Visual Studio 2022 v17.9.4 (latest)

All is reproducible at build Windows x64

To reproduce "Windows SDKs conflict" you can install standalone Windows 8.1 SDK from here:
https://go.microsoft.com/fwlink/p/?LinkId=323507

and check only install first item:

![sdk](https://github.com/xbmc/xbmc/assets/58434170/d6d73bfb-b94f-48e4-a814-2aa89438a4f7)